LSP.sublime-settings

```diff
@@ -56,9 +56,26 @@
   // hint: 4
   "show_diagnostics_severity_level": 4,
 
+  // Delay showing diagnostics by this many milliseconds.
+  // The delay will only kick into action when previously there were
+  // no diagnostics in the view. If there were previous diagnostics in the view,
+  // then the delay setting here is ignored and diagnostics are updated
+  // immediately.
+  "diagnostics_delay_ms": 0,
```

Comment:
I think that this should be set to some decent value by default. Unless you feel that it's not finished or something (I haven't had a chance to try it out properly yet).